### DISCUSSION TITLE
Is it possible to bulk auto-break lots of subtitle files via command line?

### DISCUSSION THREAD

**Discussion by enfolded** [SubtitleEdit/subtitleedit#6615] [Q&A] [answered]
I have several thousand srt files for which I would like to use SE's auto-break functionality, which I think does a great job.

Currently, for each of these files, I open it in SE, select all lines, hit unbreak (not sure if necessary), then auto break, then save.

Is there a way to do this for all the files at once, possibly via command line or some other means?

This is for a free educational project which I run on YouTube, and we have several thousand lectures, plus a couple hundred more every few months.

Thank you.

**Comment by niksedk** [comment, marked as answer]
Try this: Tools - Batch convert:
![image](https://user-images.githubusercontent.com/1261331/214798623-ad5e1725-500a-44e0-9d81-9d77e976dda7.png)

**Reply by enfolded** [reply]
That's perfect. Thank you very much! I never went into this tool because I assumed it was only for converting between subtitle file formats. Thanks for your quick help and the excellent software.
